content/updates/02-2020/1-optimize-weight-assets.md

```diff
@@ -0,0 +1,11 @@
+# Optimize the weight of your assets
+::: meta-data type="Improvement" features="Asset Manager" available="20th of February"
+:::
+
+Before applying transformations on your assets, we first convert your files into png to avoid any loss of image quality loss. This conversion can increase the size of your media files. If you need to reduce the size of your images, you can now use a new operation `optimize_jpeg`. This transformation operation will compress your file to a jpeg. Thanks to this new feature, you can easily optimize the weight of your new images and thus save asset storage space.
```

Comment:
```suggestion
Before applying transformations on your assets, we first convert your files into png to avoid any image quality loss. This conversion can increase the size of your media files. If you need to reduce the size of your images, you can now use a new operation `optimize_jpeg`. This transformation operation will compress your file to a jpeg. Thanks to this new feature, you can easily optimize the weight of your new images and thus save asset storage space.
```